### PR TITLE
fix: Update env vars and log filter

### LINKED ISSUE #2505: run/testing: TestCloudRunJobs failed
This test failed!

To configure my behavior, see [the Flaky Bot documentation](https://github.com/googleapis/repo-automation-bots/tree/main/packages/flakybot).

If I'm commenting on this issue too often, add the `flakybot: quiet` label and
I will stop commenting.

---

commit: 13ce354e578bf8b6f87438afbab4f86808f9bbf5
buildURL: [Build Status](https://source.cloud.google.com/results/invocations/9adf7d2e-db56-403c-8d52-4e4bcd06527b), [Sponge](http://sponge2/9adf7d2e-db56-403c-8d52-4e4bcd06527b)
status: failed
<details><summary>Test output</summary><br><pre>2022/04/07 18:52:08 Attempt #1: Running: runjobs-20220407-185050: Building image gcr.io/golang-samples-tests-6/runjobs:20220407-185050...
2022/04/07 18:52:08 Attempt #1: Executing: runjobs-20220407-185050: Building image gcr.io/golang-samples-tests-6/runjobs:20220407-185050: /tmp/google-cloud-sdk/bin/gcloud: --quiet builds submit --project golang-samples-tests-6 --pack=image=gcr.io/golang-samples-tests-6/runjobs:20220407-185050
2022/04/07 18:52:33 Attempt #1: Running: operation [deploy service] for service [grpc-ping-upstream]...
2022/04/07 18:52:33 Attempt #1: Executing: operation [deploy service] for service [grpc-ping-upstream]: /tmp/google-cloud-sdk/bin/gcloud: --quiet alpha run deploy grpc-ping-upstream-20220407-185050 --project golang-samples-tests-6 --image gcr.io/golang-samples-tests-6/grpc-ping:20220407-185050 --platform managed --region us-central1
2022/04/07 18:52:41 Attempt #1: Running: operation [get url] for service [grpc-ping-upstream]...
2022/04/07 18:52:41 Attempt #1: Executing: operation [get url] for service [grpc-ping-upstream]: /tmp/google-cloud-sdk/bin/gcloud: --quiet run services describe grpc-ping-upstream-20220407-185050 --project golang-samples-tests-6 --format value(status.url) --platform managed --region us-central1
2022/04/07 18:52:42 Attempt #1: Running: operation [deploy service] for service [grpc-ping]...
2022/04/07 18:52:42 Attempt #1: Executing: operation [deploy service] for service [grpc-ping]: /tmp/google-cloud-sdk/bin/gcloud: --quiet alpha run deploy grpc-ping-20220407-185050 --project golang-samples-tests-6 --image gcr.io/golang-samples-tests-6/grpc-ping:20220407-185050 --platform managed --region us-central1 --set-env-vars GRPC_PING_HOST=grpc-ping-upstream-20220407-185050-2pcfdjfgca-uc.a.run.app:443
2022/04/07 18:52:53 Attempt #1: Running: operation [get url] for service [grpc-ping]...
2022/04/07 18:52:53 Attempt #1: Executing: operation [get url] for service [grpc-ping]: /tmp/google-cloud-sdk/bin/gcloud: --quiet run services describe grpc-ping-20220407-185050 --project golang-samples-tests-6 --format value(status.url) --platform managed --region us-central1
2022/04/07 18:52:54 Attempt #1: Running: operation [id-token]...
2022/04/07 18:52:54 Attempt #1: Executing: operation [id-token]: /tmp/google-cloud-sdk/bin/gcloud: --quiet auth print-identity-token
2022/04/07 18:52:55 Attempt #1: Running: operation [id-token]...
2022/04/07 18:52:55 Attempt #1: Executing: operation [id-token]: /tmp/google-cloud-sdk/bin/gcloud: --quiet auth print-identity-token
2022/04/07 18:52:56 Attempt #1: Running: operation [delete service] for service [grpc-ping]...
2022/04/07 18:52:56 Attempt #1: Executing: operation [delete service] for service [grpc-ping]: /tmp/google-cloud-sdk/bin/gcloud: --quiet run services delete grpc-ping-20220407-185050 --project golang-samples-tests-6 --platform managed --region us-central1
2022/04/07 18:52:56 Attempt #1: Running: operation [deploy service] for service [sigterm-handler]...
2022/04/07 18:52:56 Attempt #1: Executing: operation [deploy service] for service [sigterm-handler]: /tmp/google-cloud-sdk/bin/gcloud: --quiet alpha run deploy sigterm-handler-20220407-185050 --project golang-samples-tests-6 --image gcr.io/golang-samples-tests-6/sigterm-handler:20220407-185050 --platform managed --region us-central1
2022/04/07 18:52:57 Attempt #1: Running: runjobs-20220407-185050: Creating Cloud Run Job...
2022/04/07 18:52:57 Attempt #1: Executing: runjobs-20220407-185050: Creating Cloud Run Job: /tmp/google-cloud-sdk/bin/gcloud: --quiet alpha run jobs create runjobs-20220407-185050 --image gcr.io/golang-samples-tests-6/runjobs:20220407-185050 --region us-central1 --project golang-samples-tests-6 --set-env-vars FAIL_RATE=0.0 --set-env-vars SLEEP_MS=10000
2022/04/07 18:52:59 Attempt #1: Running: runjobs-20220407-185050: Running cloud run job...
2022/04/07 18:52:59 Attempt #1: Executing: runjobs-20220407-185050: Running cloud run job: /tmp/google-cloud-sdk/bin/gcloud: --quiet alpha run jobs execute runjobs-20220407-185050 --wait --region us-central1 --project golang-samples-tests-6
2022/04/07 18:53:00 Attempt #1: Running: operation [delete container image] for service [grpc-ping]...
2022/04/07 18:53:00 Attempt #1: Executing: operation [delete container image] for service [grpc-ping]: /tmp/google-cloud-sdk/bin/gcloud: --quiet container images delete gcr.io/golang-samples-tests-6/grpc-ping:20220407-185050
2022/04/07 18:53:01 Attempt #1: Running: operation [deploy service] for service [diagram]...
2022/04/07 18:53:01 Attempt #1: Executing: operation [deploy service] for service [diagram]: /tmp/google-cloud-sdk/bin/gcloud: --quiet alpha run deploy diagram-20220407-185050 --project golang-samples-tests-6 --image gcr.io/golang-samples-tests-6/diagram:20220407-185050 --platform managed --region us-central1
2022/04/07 18:53:02 Attempt #1: Running: operation [delete service] for service [grpc-ping-upstream]...
2022/04/07 18:53:02 Attempt #1: Executing: operation [delete service] for service [grpc-ping-upstream]: /tmp/google-cloud-sdk/bin/gcloud: --quiet run services delete grpc-ping-upstream-20220407-185050 --project golang-samples-tests-6 --platform managed --region us-central1
    jobs.e2e_test.go:51: Failed to find log entries for job
2022/04/07 18:58:04 Attempt #1: Running: runjobs-20220407-185050: Deleting cloud run job...
2022/04/07 18:58:04 Attempt #1: Executing: runjobs-20220407-185050: Deleting cloud run job: /tmp/google-cloud-sdk/bin/gcloud: --quiet alpha run jobs delete runjobs-20220407-185050 --region us-central1 --project golang-samples-tests-6
2022/04/07 18:58:05 Attempt #1: Running: runjobs-20220407-185050: Deleting Image gcr.io/golang-samples-tests-6/runjobs:20220407-185050...
2022/04/07 18:58:05 Attempt #1: Executing: runjobs-20220407-185050: Deleting Image gcr.io/golang-samples-tests-6/runjobs:20220407-185050: /tmp/google-cloud-sdk/bin/gcloud: --quiet container images delete gcr.io/golang-samples-tests-6/runjobs:20220407-185050 --force-delete-tags
2022/04/07 18:58:07 Attempt #1: Running: operation [deploy service] for service [editor]...
2022/04/07 18:58:07 Attempt #1: Executing: operation [deploy service] for service [editor]: /tmp/google-cloud-sdk/bin/gcloud: --quiet alpha run deploy editor-20220407-185050 --project golang-samples-tests-6 --image gcr.io/golang-samples-tests-6/editor:20220407-185050 --platform managed --region us-central1 --set-env-vars EDITOR_UPSTREAM_RENDER_URL=https://renderer-20220407-185050-2pcfdjfgca-uc.a.run.app</pre></details>

### PR DESCRIPTION
**Pull request by averikitsch**
Fixes #2505

### CODE REVIEW

File: run/jobs/main.go [hunk 1]
```diff
@@ -107,3 +114,4 @@ func randomFailure(config Config) error {
 	}
 	return nil
 }
+// [END cloudrun_jobs_quickstart]
```
Comment (enocom) [inline run/jobs/main.go:117]: nit: please add a newline here

**Review by enocom — APPROVED** [review]

**Comment by averikitsch** [comment]
Failed test is [github/golang-samples/media](https://source.cloud.google.com/results/invocations/131a0036-6a33-47ff-b4e4-3fc55febf5b6/targets/github%2Fgolang-samples%2Fmedia)

**Review by iennae — APPROVED** [review]
LGTM

**Comment by averikitsch** [comment]
@codyoss @enocom can we go ahead and merge this?

**Comment by codyoss** [comment]
Sorry, still catching up after being out a couple. This LGTM, force pushing. We will be skipping some tests in a future PR to get things back to green.

**Pull request merged**